server/routerlicious/packages/protocol-definitions/src/config.ts

```diff
@@ -33,6 +57,12 @@ export interface IServiceConfiguration {
     // Summary algorithm configuration. This is sent to clients when they connect
     summary: ISummaryConfiguration;
 
+    // Deli lambda configuration
+    deli: IDeliConfiguration;
```

Comment:
These are config that will also be sent to client.  Not necessary saying we shouldn't, but have we thought thru whether it is useful to have client knowing these parameters, or would there be any concern that they would misuse them?

Reply:
I can't think of a misuse scenario but in the future new configs could be added, so it might not be great if this keeps getting sent to the client. Also it would be increasing the connect_document_response size unnecessarily (assuming the client doesn't care about these things). 

Would it be worth splitting this config up, so there could be an `IServiceConfiguration` that includes an `IClientConfiguration` and `IServerConfiguration` and we only send the client the `IClientConfiguration` part?

Reply:
I split them up